shacl/register.ttl

```diff
@@ -17,18 +17,18 @@
 
 reg:DatacatalogShape
     a sh:NodeShape ;
-    sh:name "Datacatalogus"@nl, "Data catalog"@en ;
```

Comment:
why no sh:name here?

Reply:
As you can see above, in this PR’s description: 

> Only use name for PropertyShape, because they are not allowed on
NodeShape.